steam_status_bot.py

```diff
@@ -103,16 +104,23 @@ def configure_bot(self):
                 SteamStatusFinder(configfile=self._configfile)
         self.user_status = self.steamstatusfinder.get_is_playing()
 
-        def _callback_status(context):
-            print('running job queue...')
+        r6s_server_poller = make_rainbow_six_siege_poller(print_on_first_run=True)
 
-            do_display, msg = self.define_job_queue()
+        self.status_checkers = [
+            self.define_job_queue,
+            r6s_server_poller
```

Comment:
Should be put into the job queue definition at self.define_job_queue and not here.